content/python/concepts/iterators/terms/iter.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: 'iter()'
+Description: 'The iter() method is a python built-in function that returns an iterator object for a given argument.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Iterators'
+  - 'OOP'
+  - 'Loops'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+The `iter()` method is a python built-in function that returns an iterator object for a given `object`.
```

Comment:
```suggestion
The **`iter()`** method is a python built-in function that returns an iterator object for a given `object`.
```

Reply:
@SSwiniarski  updated as requested adding bold for iter()

... must admit I'm a little lost with branch and github.
Everything seemed so easy in the tutorial ...

Reply:
@mcuocolo, one of the Codecademy team will come by this entry and give a second review. You don't have to do anything.